src/components/widgets/EventList/EventList.tsx

```diff
@@ -80,14 +85,14 @@ const EventList: FC<EventListProps> = () => {
         <div className={classes.component}>
             <h3>Event List</h3>
             <ul>
-                {data.events.map(({ _id, description, name, startDate, endDate, location }: IEvent) => (
+                {data.events.map(({ _id, description, name, startDate, endDate, location, image }: IEvent) => (
                     <li key={_id} className={classes.item}>
                         <h3>
                             <Link className={classes.edit} href={`/events/${_id}`}>
                                 {name}
                             </Link>
                         </h3>
-
+                        <Image src={image || ""} width={100} height={100} alt={name} priority />
```

Comment:
```suggestion
                        {image && <Image src={image} width={100} height={100} alt={name} priority />}
```
а так будет работать?

Reply:
Так тоже будет работать :) Внес изменения.